tests/integration/relations/test_database.py

```diff
@@ -57,11 +57,11 @@ async def test_build_and_deploy(ops_test: OpsTest):
     # Reduce the update_status frequency until the cluster is deployed
     async with ops_test.fast_forward("60s"):
         await ops_test.model.block_until(
-            lambda: len(ops_test.model.applications[DATABASE_APP_NAME].units) == 3
+            lambda: len(ops_test.model.applications[DATABASE_APP_NAME].units) == 1
```

Comment:
I think we can revert to 3 units in juju 3.1.6

Reply:
My bad, thanks for noticing